lib/SyTest/HomeserverFactory/Synapse.pm

```diff
@@ -102,6 +102,15 @@ sub create_server
    return $self->{impl}->new( %params );
 }
 
+sub get_critical_tests
+{
+   return (
+       'POST [^ ]+ register can create a user',
+       'POST /createRoom makes a public room',
+       'POST /rooms/:room_id/join can join a room',
+       'POST /rooms/:room_id/leave can leave a room'
```

Comment:
prolly nice to have trailing comma here

Reply:
Oh, another thing I didn't know perl could do!